app/styleguide/wskComponentHandler.js

```diff
@@ -75,6 +75,11 @@ var componentHandler = (function() {
       var registeredClass = findRegisteredClass_(jsClass);
       if (registeredClass) {
         createdComponents_.push(new registeredClass.classConstructor(element));
+        // Call any callbacks the user has registered with this component type.
+        var length = registeredClass.callbacks.length;
+        for (var n = 0; n < length; n++) {
+          registeredClass.callbacks[n](element);
+        }
```

Comment:
Bad example I picked from a quick Google for JSperf... But after rewriting it,  it is still slower (just remade to test without function call and the result was the same %). I am curious why this needs to be changed as the code is perfectly readable as is. Apologies if I am missing something obvious here... but if anything for each adds a function call which makes it more confusing to me from a readability perspective than simple for loop which everyone uses regularly. 

In addition I am following the Google Style guide which recommends using normal for loops with array based data.